src/styles/tailwind.css

```diff
@@ -0,0 +1,17 @@
+/* Import Tailwind's base styles from ui-styling package. Optional if you don't want to use them.**/
+@import '../../node_modules/@sanger/ui-styling/tailwind.css';
+
+/* Define your custom styles */
+.success-style {
```

Comment:
is the general practice to add `-style` to the end?

Reply:
Also are these needed if we only use them in 1 or two places?

Reply:
i have just kept two , in case it can be reused later. Changed names to 'success-icon', 'success-message','failure-icon', 'failure-message'